packages/typegpu/src/data/unstruct.ts

```diff
@@ -27,33 +26,32 @@ import type { BaseData } from './wgslTypes.js';
  */
 export function unstruct<TProps extends Record<string, BaseData>>(
   properties: TProps,
-): Unstruct<TProps> {
-  return new UnstructImpl(properties as TProps);
+): Unstruct<Prettify<TProps>> {
```

Comment:
Prettify here, but not on the Unstruct type. If we complete it there, we can remove it from here.